StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Components/Toast/LinkToast.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  LinkToast.swift
+//  StripePaymentSheet
+//
+//  Created by Ramon Torres on 11/23/21.
+//  Copyright © 2021 Stripe, Inc. All rights reserved.
+//
+
+import UIKit
+
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+
+/// A view for displaying a brief message to the user.
+/// For internal SDK use only
+@objc(STP_Internal_LinkToast)
+final class LinkToast: UIView {
+    struct Constants {
+        static let padding: CGFloat = 12
+        /// Space between the icon and label.
+        static let spacing: CGFloat = 8
+        static let animationDuration: TimeInterval = 0.2
+        static let animationTravelDistance: CGFloat = 40
+        static let defaultDuration: TimeInterval = 2
+    }
+
+    enum ToastType {
+        case success
+    }
```

Comment:
This is odd -- an enum with a single case?  I'm fine to leave as is, this doesn't seem needed.

Reply:
That... is weird. I think there should be an error toast too, I'm fine to leave this in for now.